Makefile

```diff
@@ -20,6 +21,9 @@ test-unit:
 test-integration-env:
 	$(PHPUNIT) --testsuite integration
 
+###
+# internal tooling rules
```

Comment:
This header should be a bit further up: `devbuild`, `prodbuild` and `test-*` are also internal tooling rules